shallow_backup/reinstall.py

```diff
@@ -64,6 +61,10 @@ def reinstall_packages_sb(packages_path):
 	"""
 	Reinstall all packages from the files in backup/installs.
 	"""
+	if not os.path.isdir(packages_path) or not os.listdir(packages_path):
```

Comment:
One last change and then I'll merge this in:

Let's refactor this out into a method and add it to all reinstallation methods.

Reply:
badabing